Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM zooniverse/ruby:jruby-1.7.18
+FROM zooniverse/ruby:2.1.5
```

Comment:
Whoops I was trying to chase down an MRI specific bug yesterday. I'll
revert.
On Jan 27, 2015 4:52 AM, "Campbell Allen" <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/zooniverse/Panoptes/pull/418#discussion_r23599972>:
>
> > @@ -1,4 +1,4 @@
> > -FROM zooniverse/ruby:jruby-1.7.18
> > +FROM zooniverse/ruby:2.1.5
>
> Any reason for switching to MRI ruby on deploy?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zooniverse/Panoptes/pull/418/files#r23599972>.
>